bouncer/util.py

```diff
@@ -70,4 +101,35 @@ def parse_document(document):
             _("The annotation has an invalid document URI"),
             "uri_not_a_string")
 
-    return (annotation_id, document_uri)
+    return {
+            "annotation_id": annotation_id,
+            "document_uri": document_uri,
+            "can_reveal_metadata": can_reveal_metadata,
+            "quote": quote,
+            "text": text
+            }
+
+
+def make_pretty_url(url):
+    """
+    Return the domain name from `url` for display.
+    """
+    try:
+        parsed_url = parse.urlparse(url)
+        pretty_url = parsed_url.netloc[:NETLOC_MAX_LENGTH]
+        if len(parsed_url.netloc) > NETLOC_MAX_LENGTH:
+            pretty_url = pretty_url + jinja2.Markup("&hellip;")
```

Comment:
The `return pretty_url` a few lines below should probably be moved here at the end of the `try` block, for better readability.